src/Build/Instance/TaskFactories/TaskHostTask.cs

```diff
@@ -28,6 +28,12 @@ namespace Microsoft.Build.BackEnd
     /// </summary>
     internal class TaskHostTask : IGeneratedTask, ICancelableTask, INodePacketFactory, INodePacketHandler
     {
+        private const int HANDSHAKE_OPTIONS_BITS = 9;
```

Comment:
maybe this should go to CommunicationsUtilities?

Reply:
Sounds logical. But this should go to some class in that namespace. Since HandshakeOptions is enum, couldn't move the consts there. Also didn't want to create a new class for that. 
Given that, I think TaskHostTask is a good place since the consts are used there.